cmd/start.go

```diff
@@ -21,7 +22,9 @@ var (
 	nodeMetricsURL string
 	id             string
 	lbBaseURL      string
+	lbTunnelURL    string
```

Comment:
eh? Ain't this suppose to be obtained from register endpoint. Whatever we do, we should have only one endpoint for load balancer in daemon

Reply:
Well, that should be added then.
Currently this is the 5223 port that the tunnel is on

Reply:
let's add that before merging this

Reply:
cc @MakMuftic ^^

Reply:
This is now ready